docs/source/releases/v1.5.rst

```diff
@@ -88,6 +88,9 @@ Minor changes
  - Changed product alerts to be sent using the Communication Event framework.
    The old email templates will continue to be supported until Oscar 2.0.
 
+ - Product reviews are no longer kept without products and being removed on
```

Comment:
This reads a bit strangely. Suggest "Product reviews are now deleted if the associated user is deleted."